Userland/Libraries/LibAudio/WavWriter.h

```diff
@@ -26,8 +27,8 @@ class WavWriter {
     WavWriter(int sample_rate = 44100, u16 num_channels = 2, PcmSampleFormat sample_format = PcmSampleFormat::Int16);
     ~WavWriter();
 
-    ErrorOr<void> write_samples(Span<Sample> samples);
-    void finalize(); // You can finalize manually or let the destructor do it.
+    virtual ErrorOr<void> write_samples(Span<Sample> samples) override;
+    virtual ErrorOr<void> finalize() override; // You can finalize manually or let the destructor do it.
```

Comment:
> `// You can finalize manually or let the destructor do it.`

Is that something that we really want? It's sad to not use RAII but panicking on IO error is not very nice either.

Reply:
> I think that this PR does too much for its own good.

no problem, I just need an explicit maintainer complaint and I can split out the CRC16 commits. However those have already been reviewed here so some context might get lost.

Reply:
You can remove this comment now